src/history.rs

```diff
@@ -0,0 +1,242 @@
+use anyhow::Result;
+use chrono::{DateTime, Local};
+use csv::*;
+use num::BigUint;
+use serde::{Serialize, Serializer};
+use std::fs::{File, OpenOptions};
+use std::path::Path;
+
+#[derive(Debug, Copy, Clone, Serialize)]
+enum Symbol {
+    BTC,
+    DAI,
+}
```

Comment:
:upside_down_face: Since they're enum variants I would use PascalCase. You can then use `#[serde(rename_all = "UPPERCASE")]` to control the serialization.

Reply:
ah good point! thanks.